src/pvrclient-nextpvr.cpp

```diff
@@ -1020,11 +1028,11 @@ PVR_ERROR cPVRClientNextPVR::GetCapabilities(kodi::addon::PVRCapabilities& capab
   kodi::Log(ADDON_LOG_DEBUG, "->GetCapabilities()");
 
   capabilities.SetSupportsEPG(true);
-  capabilities.SetSupportsRecordings(true);
-  capabilities.SetSupportsRecordingsDelete(true);
+  capabilities.SetSupportsRecordings(m_settings->m_accessLevel > 0);
+  capabilities.SetSupportsRecordingsDelete(m_settings->m_accessLevel > 1);
   capabilities.SetSupportsRecordingsUndelete(false);
   capabilities.SetSupportsRecordingSize(m_settings->m_showRecordingSize);
-  capabilities.SetSupportsTimers(true);
+  capabilities.SetSupportsTimers(m_settings->m_accessLevel == 3);
```

Comment:
Plese avoid the use of literal numbers. Define a value to make the code more readable.